### PR TITLE
Enable Mypy Check in torch/_inductor/virtualized.py

### PR DESCRIPTION
**Pull request by fffrog**
Fixes #105230

```shell
$ lintrunner init && lintrunner -a torch/_inductor/virtualized.py
...
ok No lint issues.
Successfully applied all patches.
```

```shell
$ mypy torch/_inductor/virtualized.py
Success: no issues found in 1 source file
```

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @ngimel @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov

### CODE REVIEW

File: torch/_inductor/virtualized.py [hunk 1]
```diff
@@ -259,7 +259,7 @@ class _V:
     set_interpreter_handler = _interpreter._set_handler
 
     @property
-    def ops(self) -> MockHandler:
+    def ops(self):
```
Comment (eellison) [inline torch/_inductor/virtualized.py:262]: what was the error here ?

Comment (FFFrog) [reply torch/_inductor/virtualized.py:262]: like this

```shell
torch/_inductor/virtualized.py:262:22: error: Variable "torch._inductor.virtualized._V.MockHandler" is not valid as a type  [valid-type]
torch/_inductor/virtualized.py:262:22: note: See https://mypy.readthedocs.io/en/stable/common_issues.html#variables-vs-type-aliases
Found 1 error in 1 file (checked 1 source file)
```

Now, I just ignore thie erro

```shell
    @property
    def ops(self) -> MockHandler:  # type: ignore[valid-type]
        """The operator handler specific to the current codegen task"""
        return _ops._get_handler()
```

**Review by eellison — APPROVED** [review]

**Pull request closed without merging**